src/DIPS.Xamarin.UI/Internal/Xaml/SheetView.xaml

```diff
@@ -95,7 +95,15 @@
                             Command="{Binding CancelCommand}"
                             Text="{Binding CancelTitle}"
                             Clicked="CancelButtonClicked"
-                            BackgroundColor="{Binding HeaderColor}" />
+                            BackgroundColor="{Binding HeaderColor}"
+                            HorizontalOptions="Start">
+                        <Button.Margin>

```

Comment:
I pulled the branch and tried this locally. When I set the titles to a longer string it seems to not work as good. Please test with longer strings and see how it behaves.

Reply:
Replaced with custom renderers for fixing text alignment in sheet view button.

iPhone 11 simulator - short text
https://user-images.githubusercontent.com/73521774/98635757-8ec92b80-234b-11eb-92fd-6b141b6f5b4f.png

iPhone 11 simulator - long text
https://user-images.githubusercontent.com/73521774/98635785-98529380-234b-11eb-8d57-46050b404ec6.png

OPPO F11 Pro - short text 
https://user-images.githubusercontent.com/73521774/98635774-92f54900-234b-11eb-9a8f-6371b61d4d0e.png

OPPO F11 Pro - long text 
https://user-images.githubusercontent.com/73521774/98635782-95f03980-234b-11eb-9455-15294fceaf88.png